src/store/ethereum/types.ts

```diff
@@ -26,7 +27,7 @@ export interface EthereumState extends EthereumConfig {
   // see type provider in web3-provider
   provider: provider | null
   address: string
-  signer: ethers.Signer | null
+  signer: ethers.Signer | Connector | null
```

Comment:
Don't think we should do this. If their `Connector` is a provider we should `getMetamaskSigner()`, if not we have to figure out a way to turn it into a provider.